ocp_diagnosis.sh

```diff
@@ -160,7 +134,9 @@ function store() {
 		set_pbench;
 		$1;
 	elif [[ $STORAGE_MODE == "data_server" ]]; then
-		$1 && validate_server_is_up && post_to_server "$OUTPUT_DIR/$2"
+		snappy login
```

Comment:
Quick question, where are we passing the credentials for login?

Reply:
@mfleader we need to move this https://github.com/mfleader/snappyCLI to openshift-scale too
@chaitanyaenr I think there will be a prompt to enter username:password
will later test it out with Matt

Reply:
With no options passed, `login` will prompt you at the command line for your credentials. 

Additionally, you can pass your username as an environment variable or as an option. Password can also be passed as a cleartext option.

```shell
Usage: snappy login [OPTIONS] [URL]
Arguments:
  [URL]  [env var: DATA_SERVER_URL;default: http://localhost:8000]

Options:
  --username TEXT  [env var: DATA_SERVER_USERNAME; required]
  --password TEXT  [required]
```

Reply:
Shouldn't we pass the username/password in an automated way in this case?

Reply:
Maybe add the DATA_SERVER_USERNAME and DATA_SERVER_PASSWORD options in https://github.com/openshift-scale/scale-ci-diagnosis/blob/master/env.sh since we source this file before running scale-ci-diagnosis? Also a check to see if they are set would be useful IMHO.

Reply:
you sure you want folks leaving passwords in cleartext scripts?

Reply:
Right, it's a bad idea to leave the password in the file. Let's just add checks to see if the username and password are set as environment variables and let user know if they are not set before trying to login. Thoughts?

Reply:
I added a `script-login` function to script the login command with environment variables with the names mentioned above. You can see it in the [snappycli readme](https://github.com/mfleader/snappyCLI) under 'Help with a command'.